README.md

```diff
@@ -5,13 +5,12 @@ Node.js API**  also known as **N-API**, providing C++ object model and exception
 handling semantics with low overhead. It guarantees backward compatibility with
 use with older versions of Node.js that do not have N-API built-in.
 
```

Comment:
How about rearranging the text like this:

This package contains **header-only C++ wrapper classes** for the **ABI-stable Node.js API** also known as **N-API**, providing C++ object model and exception handling semantics with low overhead. It guarantees backward compatibility when  used with older versions of Node.js that do not have N-API built-in.
N-API is an API for building native addons. It is independent from the underlying JavaScript runtime (e.g. V8 or ChakraCore) and is maintained as part of Node.js itself. This API will be Application Binary Interface (ABI) stable across versions and flavors of Node.js. It is intended to insulate native addons from changes in the underlying JavaScript engine and allow modules compiled for one version to run on later versions of Node.js without recompilation. N-API guarantees the API and ABI compatibility across different versions of Node.js. So if you switched to a different version of Node.js, you would not need to reinstall or recompile the native addon.